### PR TITLE
Regular Key Signing

### PR DESCRIPTION
**Pull request by vhpoet**
- Add regular key generation
- Add regular-wallet export functionality
- Use regular key for signing transactions when using the regular-wallet

### CODE REVIEW

**Comment by vhpoet** [comment]
@MatthewPhinney it's the same code that generates the original masterkey.

File: wallet-regular [hunk 1]
```diff
@@ -0,0 +1 @@
+eyJpdiI6InBJUkFycUtZQ1Y4Q1lla0Q3TkJQZlE9PSIsInYiOjEsIml0ZXIiOjEwMDAsImtzIjoyNTYsInRzIjo2NCwibW9kZSI6ImNjbSIsImFkYXRhIjoiIiwiY2lwaGVyIjoiYWVzIiwic2FsdCI6IkE4QitDQTFURFBjPSIsImN0IjoiOWphdnRURER1S3QveFBtc00rc1RiY2pjSU5hZFI5QlBMMmp6VkYwaXZpNmFWdnZ2a2hBTjdQY25JZk13T2FzcTU1SlFQOVl3QlRaMzR5cDFuNGRrRXN6RmUwdW1vaGthbnRLVjdwbENEVmJEQ1VMZ1lqaHpHUU9zV2lubEc0ZGNBcHVJWHpielBteEJqZlBUS2RGT0I1QytGVStRTlVSV20vUlhEOHR2SXo1MUh4SWFyaWhTU2RJRWp3Z01XZWNTZzFhWFczRTBkeEFjV1J3Q3FzQUpYMldUb2wrbmFqcnpUZUpTSTJBRW1TaUc5djFxTzRvOVhnOGI4TFV0WlJtQUV3TSs0R01Nd0p3TTViVWZUV29GMEVkZGwrMXBhL2Nyd3hpejR2cnRSQUs1WGRRQzZBbXArVmkyTVlvc3FNN1QxU1kyeHZzdTRuWjZRTlhTM1pHcFk0T01BSW82U2YrQy9jQjh0RUVtcXBhN0s0Y1p1bGtPS2pIN21oUWEvVTZGZmxRbnFYblcvYkh3RDlWa0tqdUNZTk5wZVRlQnBDd0RaVWNvYjliVWNnSitzbzFnL2ZTOFFEaWwyODhGN3ZndXNsaGthdnphREl0cFozeFgwK1VPTm84MHVQbm91WHRoVVdIZDF0QjJ6bnEzRTJvWVVodnRNMzRhc3VpV0FVSUNJVDJlWm56V0MrdmVFck9pT0xkOFgvUzJZSm5ML2xxRmRadzRqQ2o3NkdvQ3hxemtPZEs4amwxVmYzU2cxTHpVd2Q1SUo2RVFFMDNkaDl1TWdscWNTVG1WSUlpZUdoVkw1ZEVibDB0VFp4WUdsSWs9In0=
```
Comment (MatthewPhinney) [inline wallet-regular]: Don't need this file.

Comment (vhpoet) [inline wallet-regular]: ah sorry

**Comment by mrajvanshy** [comment]
LGTM

**Pull request merged**